charts/zabbix/values.yaml

```diff
@@ -343,8 +361,12 @@ zabbixAgent:
   extraVolumeMounts: []
   # -- annotations to add to the daemonSet
   daemonSetAnnotations: {}
+  # -- labels to add to the daemonSet
+  daemonSetLabels: {}
```

Comment:
Hi @syphernl!

Could you create the deploymentLabel to use in the Zabbix Server and Zabbix Proxy template when the ZabbixAgent is not in daemonSet mode?

```
# -- labels to add to the deploymentSet
deploymentLabels: {}
```